weave-js/src/components/Icon/Icon.tsx

```diff
@@ -291,6 +291,10 @@ const updateIconProps = (props: SVGIconProps) => {
     width: 20,
     height: 20,
     ...props,
+    style: {
+      ...props.style,
```

Comment:
I don't really think it will come up, but do you think props.style should take precedence?

Reply:
hmm fair point. that would align with the current expectations for weave components so i'll update to make it an overrideable default